lib/replicache.dart

```diff
@@ -323,8 +323,9 @@ class Replicache implements ReadTransaction {
 
   static Future<dynamic> _invoke(String dbName, String rpc,
       [dynamic args = const {}]) async {
+    final enc = JsonUtf8Encoder();
```

Comment:
Why does the Android version do JSON encoding, but the iOS one doesn't?

Reply:
Not sure what you mean?

This is the Dart code. It used to send a `String`. Now it sends a `Uint8List` instead. Both Java and ObjC just pass this along without any copying.

Reply:
Oh nice, I misunderstood, apologies.